lib-android/src/main/java/com/devinci/lib/view/IconTabLayout.java

```diff
@@ -0,0 +1,54 @@
+package com.devinci.lib.view;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.TabLayout;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v7.widget.Toolbar;
+import android.util.AttributeSet;
+import com.devinci.lib.R;
+
+public class IconTabLayout extends TabLayout {
```

Comment:
Consider name change. TabLayout already supports icon setting, this class on the other hand adds support for getting those icons from LayoutViewPager.